src/dotnet/Chat.Client/ClientDefs.cs

```diff
@@ -54,6 +54,8 @@ public interface IChatsClientDef
     [Get(nameof(FindNext))]
     Task<ChatEntry?> FindNext(Session session, string chatId, long? startEntryId, string text, CancellationToken cancellationToken);
 
+    [Post(nameof(AddMembers))]
+    Task AddMembers([Body] IChats.AddMembersCommand command, CancellationToken cancellationToken);
```

Comment:
There is `CreateAuthorsCommand` at `IAuthors` - highly likely it does the same.
Please have a look.